src/main/java/core/basesyntax/service/impl/operation/BalanceOperation.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.impl.operation;
+
+import core.basesyntax.dao.FruitStorageDao;
+
+public class BalanceOperation implements OperationHandler {
+    private final FruitStorageDao storageDao;
+
+    public BalanceOperation(FruitStorageDao storageDao) {
+        this.storageDao = storageDao;
+    }
+
+    @Override
+    public void doOperation(String fruitName, Integer quantity) {
+        storageDao.setQuantity(fruitName, quantity);
```

Comment:
What happens if the number is negative?)
Let's change it and add tests for it. See if other handlers need it too

Reply:
changed